frontend/model/state.js

```diff
@@ -310,6 +315,12 @@ const getters = {
       })()
     }
   },
+  userDisplayName (state, getters) {
+    return (username) => {
+      const profile = getters.globalProfile(username) || {}
+      return profile.displayName || profile.name || username
```

Comment:
`profile.name` is the same as `username` (and we should open an issue to rename it to `username` remove it entirely... ok done: #848), so the check for `profile.name` is not needed, and since we might remove that property later it should be removed.

Reply:
Checking `profile.name || username`, is needed in case `profile` returns an empty object (a.k.a. this user is not part of the group anymore).

Reply:
I don't think that's correct. Even if they aren't part of this group anymore, you're passing in `username` into this function. So it's presumed you already have it, therefore you don't need to check `profile.name` which is identical to `username`

Reply:
Oh I see, changed!